clients/shared/table_config.go

```diff
@@ -27,38 +31,16 @@ type GetTableCfgArgs struct {
 	ColumnTypeLabel string
 	// Description of the column (used to annotate whether we need to backfill or not)
 	ColumnDescLabel    string
-	EmptyCommentValue  *string
 	DropDeletedColumns bool
 }
 
-func (g GetTableCfgArgs) ShouldParseComment(comment string) bool {
-	if g.EmptyCommentValue != nil && comment == *g.EmptyCommentValue {
-		return false
-	}
-
-	// Snowflake and Redshift both will return `<nil>` if the comment does not exist, this will check the value.
-	// BigQuery returns ""
-	return true
-}
-
 func (g GetTableCfgArgs) GetTableConfig() (*types.DwhTableConfig, error) {
-	// Check if it already exists in cache
-	tableConfig := g.ConfigMap.TableConfig(g.TableID)
-	if tableConfig != nil {
+	if tableConfig := g.ConfigMap.TableConfig(g.TableID); tableConfig != nil {
 		return tableConfig, nil
 	}
 
-	rows, err := g.Dwh.Query(g.Query, g.Args...)
-	defer func() {
-		if rows != nil {
-			err = rows.Close()
```

Comment:
From Golang:

```go
// Close closes the [Rows], preventing further enumeration. If [Rows.Next] is called
// and returns false and there are no further result sets,
// the [Rows] are closed automatically and it will suffice to check the
// result of [Rows.Err]. Close is idempotent and does not affect the result of [Rows.Err].
```